packages/types/src/legacy/reference-config.ts

```diff
@@ -1,11 +1,160 @@
-import type { ExternalDocumentation } from '@/entities/workspace/collection'
-import type { RequestMethod } from '@/helpers'
-import type { OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from '@scalar/openapi-parser'
-import type { HarRequest } from 'httpsnippet-lite'
+import type { ThemeId } from '@scalar/themes'
+import type { UseSeoMetaInput } from '@unhead/schema'
+import type { HarRequest, TargetId } from 'httpsnippet-lite'
+import type { OpenAPIV2, OpenAPIV3, OpenAPIV3_1 } from 'openapi-types'
 
+/**
+ * @deprecated This type does not provide much type safety, we shouldn't really use it
```

Comment:
But this is exactly what it’s for: For objects where we don’t know anything about the shape. 🤔

Reply:
Probably `Record<string, unknown>` would be a better choice